src/main/pages/setup-docker/docker-config.adoc

```diff
@@ -77,14 +78,14 @@ Instance configuration is generated by Che and is updated by our internal config
 ----
 
 [id="jdbc-configuration"]
-== JDBC Configuration
+== Configuring JDBC
 
-Eclipse Che uses http://www.h2database.com/html/main.html[H2] for single-user builds and PostgreSQL database in a multi-user flavor.
+Eclipse Che uses the http://www.h2database.com/html/main.html[H2] database for single-user builds and PostgreSQL database in multi-user mode.
 
-Depending on the used database, JDBC Connection pool will be initialized with respective default values. These values can be overridden through che.env.
+The JDBC Connection pool is initialized with respective default values. These values can be overridden in the `che.env` file.
```

Comment:
Suggest retaining 'Depending on the used database,' from the original, makes it clear.